test/system/tests/tcti/abrmd/extended-sessions.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
```

Comment:
The other tests scripts uses a different shebang. I don't really mind which one to use but I prefer if we keep it consistent.

Also the other tests have a header with Copyright, etc. I would also like it to have consistency with that.

Reply:
Yeah their is a lot-of-cleanups to do in these patches, and it will match the others we have already established.